engine/evarutil.mli

```diff
@@ -273,3 +258,27 @@ type type_constraint = types option
 [@@ocaml.deprecated "use the version in Evardefine"]
 type val_constraint = constr option
 [@@ocaml.deprecated "use the version in Evardefine"]
+
+
+(** e_* stuff *)
```

Comment:
Please explain what "e_ stuff" is (eg when a programmer should use these functions)

Reply:
>Please explain what "e_ stuff" is (eg when a programmer should use these functions)

Never, they're deprecated.
I removed the comment since it does nothing of interest.